dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -36,22 +36,22 @@
 # # {3} The error message from the exception
 ANNO_CLASSSOURCE_CLOSE1_EXCEPTION=CWWKC0001W: [ {0} ]: The close of source [{1}] [{2}] failed. Error message: {3}
 ANNO_CLASSSOURCE_CLOSE1_EXCEPTION.explanation=The close for the resource caused an exception.
-ANNO_CLASSSOURCE_CLOSE1_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_CLOSE1_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
 
 # # {0} A hashcode id used to identify the object emitting the message
 # # {1} The name of the resource
 # # {2} the name of the class associated with the resource
 # # {3} The error message from the exception
 ANNO_CLASSSOURCE_CLOSE2_EXCEPTION=CWWKC0002W: [ {0} ]: The close of resource [{1}] for class [{2}] failed. Error message: {3}
 ANNO_CLASSSOURCE_CLOSE2_EXCEPTION.explanation=The close of the input stream for the resource caused an exception.
-ANNO_CLASSSOURCE_CLOSE2_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_CLOSE2_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
 
 # # {0} A hashcode id used to identify the object emitting the message
 # # {1} The name of the resource, includes the full path to the class
 # # {2} the name of the class associated with the resource
 ANNO_CLASSSOURCE_OPEN1_EXCEPTION=CWWKC0003W: [ {0} ]: The open of resource [{1}] for class [{2}] failed.
 ANNO_CLASSSOURCE_OPEN1_EXCEPTION.explanation=Opening an InputStream for the resource caused an exception.
-ANNO_CLASSSOURCE_OPEN1_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_OPEN1_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
```

Comment:
Change
'For more information on the topic, see the exception and the associated FFDC message.'
to
'For more information, see the exception and the associated FFDC message.'

Reply:
Updated.